angular/readme.md

```diff
@@ -35,13 +35,15 @@ export const clientId = '{CLIENT_ID}';
 Add the following to your **Allowed Callback URLs** settings:
 
 ```text
-com.auth0.samples://{DOMAIN}/capacitor/com.auth0.samples/callback
+com.auth0.samples://{DOMAIN}/capacitor/com.auth0.samples/callback, 
+http://localhost:4200
```

Comment:
Can we do something comparable as to what's done below?

If running on a mobile device, add the following to your **Allowed Callback URLs**  settings:

```text
com.auth0.samples://{DOMAIN}/capacitor/com.auth0.samples/callback
```

If running on a desktop browser, add the following to your **Allowed Callback URLs** settings:

```text
http://localhost:4200
```